src/isPalindrome.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+export default function isPalindrome(str) {
```

Comment:
I would avoid variable names like `str`. instead be more verbose. short or truncated variables names are generally harder to understand.